mapper/mapper.go

```diff
@@ -441,9 +441,14 @@ func getWindowOffset(windowSize int) int {
 	return windowSize / 2
 }
 
-func calendarEntryFromRelease(release search.Release) model.CalendarEntry {
+func calendarEntryFromRelease(release search.Release, uriPrivatePrefix string) model.CalendarEntry {
 	result := model.CalendarEntry{
-		URI:         release.URI,
+		URI: func(uri, privateSuffix string) string {
+			if privateSuffix == "" {
+				return release.URI
+			}
+			return privateSuffix + release.URI
```

Comment:
should we ensure that there is a slash here, i.e. release.URI always starts with /

Reply:
yes I think that's a good idea. I always assumed we would have a path starting with a '/', but no harm checking and adding one if it's not there

Reply:
Hi Steve added the check into the config validation (only done once then but may be overkill?), have a look